patchwork-extensions-blockentity/src/main/java/net/minecraftforge/common/extensions/IForgeTileEntity.java

```diff
@@ -171,30 +173,4 @@ default boolean canRenderBreaking() {
 	default boolean hasFastRenderer() {
 		return false;
 	}
-
-	/**
-	 * Requests a refresh for the model data of your TE
-	 * Call this every time your {@link #getModelData()} changes.
-	 */
-	default void requestModelDataUpdate() {
-		BlockEntity te = getTileEntity();
-		World world = te.getWorld();
-
-		if (world != null && world.isClient) {
-		//	ModelDataManager.requestModelDataRefresh(te);
-		}
-	}
-
-	/**
-	 * Allows you to return additional model data.
-	 * This data can be used to provide additional functionality in your {@link net.minecraft.client.renderer.model.IBakedModel}
-	 * You need to schedule a refresh of you model data via {@link #requestModelDataUpdate()} if the result of this function changes.
-	 * <b>Note that this method may be called on a chunk render thread instead of the main client thread</b>
-	 *
-	 * @return Your model data
-	 */
-	//@Nonnull
-	//default IModelData getModelData() {
-	//	return EmptyModelData.INSTANCE;
```

Comment:
Is this method not needed?

Reply:
It is declared in one of our other classes

Reply:
Need to do this because we want to keep things modular and also helps resolving a circular dependency problem

Reply:
:+1: